src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -361,6 +361,17 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
                 return Promise.resolve();
             }));
 
+        // if table have column with generated type, we must add the expression to the meta table
+        await Promise.all(table.columns
+            .filter(column => column.generatedType === "STORED" && column.asExpression)
+            .map(async column => {
+                const tableName = await this.getTableNameWithSchema(table.name);
+                const deleteQuery = new Query(`DELETE FROM typeorm_generation_meta WHERE table_name = $1 AND column_name = $2`, [tableName, column.name]);
```

Comment:
I'm not sure if you aware or not, but we already have a `typeorm_metadata` table where we store metadata... we should probably use it instead of generating of a new table....

Reply:
Otherwise PR looks good and can be merged

Reply:
> I'm not sure if you aware or not, but we already have a `typeorm_metadata` table where we store metadata... we should probably use it instead of generating of a new table....

I was aware. However, I was not able to find a usage of this table somewhere and therefore didn't know how to use it correctly. I didn't want to cause any conflicts. And it is hard to identify whether you could cause any conflicts if you don't know how it is already used.

Reply:
@pleerock I would appreciate an response :) I need to know If I need to refactor the code to use the other meta table. However, I still don't know if this meta table fits the usage.

Reply:
@pleerock Just another small reminder ;) Really want this support in typeorm.

Reply:
My response is the same - we don't want to have separate table for each feature we have inside schema sync. We should use `typeorm_metadata` table to store schema metadata information for any use case we have. You can see how its used in the same `RdbmsSchemaBuilder` file.

Reply:
But it only gets generated in RdbmsSchemaBuilder. And there is no clear documentation ? Of course `database`, `schema` and `table` are self explaining. But what is `type` and `name` used for? I don't want to override anything which may be in use. Otherwise I would do it like this:

- `type`: `generated`
- `name`: Column name
- `value`: Expression

Is this ok?

Reply:
> And there is no clear documentation ?

This stuff is internal, that's why there is no clear documentation. Also, this code wasn't written by me, so I also don't know all the details. Source code is your best friend 😆